### PR TITLE
Automatic commit: Move 'conversational-ai-sdk-ios-integrate-sampleapp…

### PR DESCRIPTION
**Pull request by MarcoEidinger**
…' from QA to Production

### CODE REVIEW

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+[ACCORDION-BEGIN [Step 6: ](Test your iOS application)]
+
+You are now ready to test your iOS application and initiate a conversation with your chatbot.
+
+1. In Xcode, press `⌘R` to run your application in the iOS simulator.
+
+2. Touch on the row in the `ODataContainers` view.
+
+    ![ODataContainers View](SampleAppODataContainers.png)
```
Comment (MichaelCzcz) [inline tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:255]: Add border.

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+3. Touch on the right bar button.
+
+    ![Collections View](SampleAppCollectionsModifiedRightBarButtonItem.png)
```
Comment (MichaelCzcz) [inline tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:259]: Add border.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Hello Marco,

thanks for creating a new tutorial. The content is looking good, except for some minor formatting issues. Some screenshots need borders to increase visibility.

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+auto_validation: true
+time: 15
+tags: [ tutorial>intermediate, software-product>sap-btp-sdk-for-ios, software-product>sap-mobile-services ]
+primary_tag: software-product>sap-conversational-ai
+author_name: Marco Eidinger
+author_profile: https://github.com/MarcoEidinger
+---
+
+## Prerequisites
+ - You have created a SAP Conversational AI chatbot, for example, as described in the tutorial [Build Your First Chatbot with SAP Conversational AI](https://bit.ly/3srXE1F).
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Create a Mobile Channel in SAP Conversational AI)]
+
+Sign-in to your bot on [https://cai.tools.sap/](https://cai.tools.sap/) and perform the following steps in the **Connect** tab:
+
+1. Click the `+` button.
+
+    ![Add SAP Conversational AI Mobile SDK channel](./connect-primaryChannel-mobilesdk-add.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:30]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+
+    A model popup opens.
+
+2. Enter `Name` and `Application ID`.
+
+    ![Add SAP Conversational AI Mobile SDK channel](./connect-primaryChannel-mobilesdk-create.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:36]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Create OAuth2 Client in SAP Conversational AI)]
+
+
+1. Access `Tokens` through your bot's `Settings`.
+
+    !["Generate OAuth client"](./connect-settings-oauthclient-generate.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:50]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+
+    ![Choose client authentication type](./connect-settings-oauthclient-generate-chooseauthtype.png)
+
+4. Close the confirmation dialog.
+
+    ![Confirmation dialog](./connect-settings-oauthclient-confirm.png)
+
+5. Copy the generated `Auth URL`, `Client ID` and `Client Secret` to reuse later.
+
+    ![Generated OAuth client](./connect-settings-oauthclient-generated.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:64]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 3: ](Create a destination in SAP Mobile Services)]
+
+1. Click on `Mobile Connectivity` in your application definition.
+
+    !["Navigate to Mobile Connectivity feature"](./ms-features.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:73]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+
+2. Click the `+` button to start the destination create wizard.
+
+    ![Create a new destination](./ms-destination-create.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:77]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 10]
```diff
@@ -0,0 +1,276 @@
...
+
+    > Note: `Token Service URL` is named `Auth URL` in SAP Conversational AI.
+
+    Click **Next**.
+
+7. Click **Finish**.
+
+8. Click `Ping` to test if the destination can reach its target.
+
+    ![Ping the target system to verify that destination works](./ms-destination-created_test.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:118]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md [hunk 11]
```diff
@@ -0,0 +1,276 @@
...
+
+[ACCORDION-BEGIN [Step 4: ](Add SAP Conversational AI SDK for iOS to your Xcode project)]
+
+1. Open your Xcode project.
+
+2. In Xcode menu select <kbd>File</kbd> >  <kbd>Add Packages...</kbd> which opens a modal dialog.
+
+3. Paste the following URL into the search field: `https://github.com/SAP/cloud-sdk-ios-cai`
+
+    ![Add Package](./xcode_addPackage.png)
```
[the remark this answers is not in the document]
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:133]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md (hunk 1 above)
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:255]: done

File: tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md (hunk 2 above)
Comment (MarcoEidinger) [reply tutorials/conversational-ai-sdk-ios-integrate-sampleapp/conversational-ai-sdk-ios-integrate-sampleapp.md:259]: done

**Comment by MarcoEidinger** [comment]
> Hello Marco,
> 
> thanks for creating a new tutorial. The content is looking good, except for some minor formatting issues. Some screenshots need borders to increase visibility.

Thanks Michael for the feedback. I applied the suggested changed :)

**Comment by MichaelCzcz** [comment]
It seems that you did not push the requested changes.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
It seems that you did not push the requested changes.

**Comment by MarcoEidinger** [comment]
@MichaelCzcz I did with https://github.com/sap-tutorials/Tutorials/pull/7358/commits/6d91f1e480892f964e3925a37e5b7677758bb83f

**Comment by MichaelCzcz** [comment]
> @MichaelCzcz I did with [6d91f1e](https://github.com/sap-tutorials/Tutorials/commit/6d91f1e480892f964e3925a37e5b7677758bb83f)

No, you didn't. That commit only contains two changes, none of them is adding a border to a screenshot.

image

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please address the requested changes.
You can find information on how to add borders to images in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64).

**Comment by MarcoEidinger** [comment]
@MichaelCzcz i previously committed the image files with a border. I wasn’t aware of the style guide, thanks for sharing! So instead of actually changing the images it is possible to add a border in markdown??? (using the !! encoding within the MD file)

**Comment by MarcoEidinger** [comment]
I added borders to images through markdown encoding !! with commit https://github.com/sap-tutorials/Tutorials/pull/7358/commits/bb5a2b15b89f1417124c26fc858a9bb9b729939c

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
I can see the changes in the code now. I am sorry to ask you to make more changes, but your screenshots are to big. So, the borders don't show.
according to the style guide the maximum size 1200x500px. Except for the screenshots taken from your mobile device, all exceed that size.
Please reduce the size of your screenshots and check if the borders are visible in the tutorial.
You can do this here: https://developers-qa.sap.com/tutorials/conversational-ai-sdk-ios-integrate-sampleapp.html
When you replace images, you also have to make a small change to the md-file. Adding a space somewhere is enough. Otherwise the new images will not be displayed.

**Comment by MarcoEidinger** [comment]
@MichaelCzcz borders are shown in preview 
Screen Shot 2022-01-04 at 8 09 44 AM but if you check https://developers-qa.sap.com/tutorials/conversational-ai-sdk-ios-integrate-sampleapp.html then you won't find my changes as I did not double-maintain them in Tutorials-Contributions yet. My amendments were done for this PR only.

Ok, if screen shots have to have a maximum size then I will reduce the size

**Comment by MarcoEidinger** [comment]
@MichaelCzcz I checked but resizing the images would lead to significant quality loss. I don't want to retake all screenshots. I'd like to appeal to get this PR merged as-is

**Comment by MarcoEidinger** [comment]
Images were resized to be smaller or equal 1200 width. I copied the changes to QA as well

Screen Shot 2022-01-04 at 11 50 24 AM

**Pull request merged**